### DISCUSSION TITLE
Button for capture (ROG Ally user)

### DISCUSSION THREAD

**Discussion by Lukierek** [unknownskl/xbox-xcloud-client#1107] [Q&A]
Thanks to @unknownskl for making this awesome peace of software. I would like say Thanks in PayPal language, but there's only MasterCard language available.  😶‍🌫️

Anyway. 

**There's a key map for "Share" button on xbox series gamepad?** 

offtop.:
Greenlight work amazing on ROG Ally. It's a most have for this small device. Definetly consider "Thanks" (in any language) for @unknownskl for making this.

**Comment by mikezue-co** [comment]
@OrionFanWeb1701 odezwij się do mnie na maila, mamy dużo zleceń dla Ciebie
